apps/partners/src/app/organization/[orgId]/hooks/components/InputValuesForm.tsx

```
@@ -0,0 +1,175 @@
+
+const parseInputValue = (value: string, type: InputValueType): InputValuesParameters[keyof InputValuesParameters] | null => {
+  try {
+    switch (type) {
+      case "number":
+        return Number(value)
+      case "boolean":
+        return value.toLowerCase() === "true"
+      case "object":
+        return JSON.parse(value)
+      case "array":
+        const parsed = JSON.parse(value)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)